cache/ttl/cache.go

```diff
@@ -110,11 +134,25 @@ func (cache *Cache) Delete(key string) bool {
 	return true
 }
 
+// Range calls f on every key
+func (cache *Cache) Range(f func(key string, value interface{}) error) {
```

Comment:
`Range` is vague here. I would prefer `EvictOnError` or `ExamByCall`

Reply:
i think `Range()` is a standard/popular name for this? similar to `range` keyword in golang
```
for k := range map
```